### ISSUE TITLE
Potential FLIC implementation

### ISSUE THREAD

**Issue by tianyuan129** [yoursunny/NDNts#26]
Just out of curiosity. Do you plan to implement [FLIC](https://datatracker.ietf.org/doc/draft-irtf-icnrg-flic/)? It will help transfer large dataset

**Comment by yoursunny** [comment]
FLIC is in the plan since 2021:
https://datatracker.ietf.org/meeting/interim-2021-icnrg-01/materials/minutes-interim-2021-icnrg-01-202112101700-06.html

**Comment by tianyuan129** [comment]
Yeah that's why I asked since I didn't see implementations three years after.
